app-backend/database/src/main/scala/com/azavea/rf/database/tables/Buckets.scala

```diff
@@ -223,6 +223,23 @@ object Buckets extends TableQuery(tag => new Buckets(tag)) with LazyLogging {
     }
   }
 
+  /** Adds a list of scenes to a bucket
+    *
+    * @param sceneIds Seq[UUID] list of primary keys of scens to add to bucket
+    * @param bucketId UUID primary key of back to add scenes to
+    */
+  def addScenesToBucket(sceneIds: Seq[UUID], bucketId: UUID)
+    (implicit database: DB): Future[Int] = {
+
+    database.db.run {
+      DBIO.sequence(sceneIds.map { sceneId =>
+        ScenesToBuckets.insertOrUpdate(SceneToBucket(sceneId, bucketId))
+      }).transactionally
```

Comment:
We use a `transactionally` for `insertOrUpdate` but not for `delete` below because each `insertOrUpdate` is a separate query, so we have a list of queries that should only run in one batch. The `delete` on the other hand is just one query, thus is atomic by definition.